src/overlays/actors/ovl_Object_Kankyo/z_object_kankyo.h

```diff
@@ -7,9 +7,24 @@ struct ObjectKankyo;
 
 typedef void (*ObjectKankyoActionFunc)(struct ObjectKankyo*, GlobalContext*);
 
+typedef struct {
+    /* 0x00 */ f32 unk_00;
+    /* 0x04 */ f32 unk_04;
+    /* 0x08 */ f32 unk_08;
+    /* 0x0C */ f32 unk_0C;
+    /* 0x10 */ f32 unk_10;
+    /* 0x14 */ f32 unk_14;
+    /* 0x18 */ f32 unk_18;
+    /* 0x1C */ u8 unk_1C;
+} ObjectKankyoStruct; // size = 0x20
+
 typedef struct ObjectKankyo {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x100C];
+    /* 0x0144 */ f32 unk_144;
+    /* 0x0148 */ f32 unk_148;
+    /* 0x014C */ ObjectKankyoStruct unk_14C[128];
```

Comment:
Could this be one of those particle structs?

Reply:
Probably, size doesn't match any of the pre-made ones though.